.circleci/config.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: cimg/base:2020.01
+    working_directory: ~/hypertrace
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: # official docs won't work, see https://github.com/circleci/circleci-docs/issues/1323#issuecomment-323620271
+          name: Install Docker Compose
+          command: |
+            set -x
+            curl -L https://github.com/docker/compose/releases/download/1.24.1/docker-compose-`uname -s`-`uname -m` > ~/docker-compose
+            chmod +x ~/docker-compose
+            sudo mv ~/docker-compose /usr/local/bin/docker-compose
+
+      - run: jq --version
+      - run: docker-compose -f docker/docker-compose.yml up -d
+      - run: curl -i http://localhost:8081
```

Comment:
probably 2020 is better as that's the hypertrace UI port.. that or parse docker-compose for healthy.. that or put a TODO to expose a easy api to fetch a trace by ID that we upload

Reply:
This curl was incorrect. I am changing it now.